### DISCUSSION TITLE
is there anyway to stop the ‘howFrequent’ to stop rounding up to 5?

### DISCUSSION THREAD

**Discussion by mwziesv** [derekantrican/GAS-ICS-Sync#459] [Q&A] [answered]
my jobs require instant update, is there anyway for me to reducing the interval to 1 minute. thank you

ps. i have no experience in coding.

**Comment by jonas0b1011001** [comment, marked as answer]
Add `1` to the list [here](https://github.com/derekantrican/GAS-ICS-Sync/blob/dd4c000080fe392fa893adb9ed5e977dbc1d0bbd/Helpers.gs#L69) and rerun install.

**Reply by mwziesv** [reply]
Thank you so much for this quick reply !
